cypress/objects/timeLogingPage.js

```diff
@@ -0,0 +1,27 @@
+class TimeLogingPage {
+    getUrl(){
+        return cy.url().should('include', '/time-logging')
+    }
+    getPageTitle(){
+        return cy.get('.page__title')
+    }
+    getCalendarVisible(){
+        return cy.get('.calendar')
+    }
+    getCalendarToday(){
+        return cy.get('.calendar--today')
+    }
+    getTileForm(){
+        return cy.get('.tile.form')
+    }
+    getInfoTitle(){
+        return cy.get('.user-info__title')
+    }
+    getLength(){
```

Comment:
This could be more descriptive, for example `getNavigationTabCount()` or something similar, because right now it's not really clear whose length you're getting :)